src/PostRector/Application/PostFileProcessor.php

```diff
@@ -27,20 +27,34 @@ final class PostFileProcessor implements ResetableInterface
      */
     private array $postRectors = [];
 
+    /**
+     * @param PostRectorInterface[] $additionalPostRectors
+     */
     public function __construct(
         private readonly Skipper $skipper,
         private readonly UseAddingPostRector $useAddingPostRector,
         private readonly NameImportingPostRector $nameImportingPostRector,
         private readonly ClassRenamingPostRector $classRenamingPostRector,
         private readonly DocblockNameImportingPostRector $docblockNameImportingPostRector,
         private readonly UnusedImportRemovingPostRector $unusedImportRemovingPostRector,
-        private readonly RenamedClassesDataCollector $renamedClassesDataCollector
+        private readonly RenamedClassesDataCollector $renamedClassesDataCollector,
+        private array $additionalPostRectors
     ) {
     }
 
     public function reset(): void
     {
         $this->postRectors = [];
+        $this->additionalPostRectors = [];
```

Comment:
When resetting the state between tests we also need to reset the list of additional post rectors